.travis.yml

```diff
@@ -45,7 +46,10 @@ before_script:
 
 script:
     - make
-    - if [ -z "$CROSS_COMPILE" ]; then make test; fi
+    - if [ -n "$CROSS_COMPILE" ]; then
+          export EXE_PRE="wine " EXE_EXT=".exe" WINEPREFIX=`pwd`;
```

Comment:
I don't think /proc/sys/fs/binfmt_misc is enabled inside the Travis build environments (they started using containers instead of full VMs). Another solution would be to change `shlib_wrap.sh` to automatically use wine but I don't know how that would work, and EXE_PRE is probably simpler anyway.